### ISSUE TITLE
[Accessibility] Error audio cues are not played on a character level.

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#172525]
CC @meganrogge 

* Notes: This is not a regression.

1. Create test.ts:

``` ts
// line 1 comment
xxxx // line 2: intentional error
// line 3 comment
```

1. With screen reader, do the following:

* In l2, use LeftArrow and RightArrow to navigate the line character by character: You only hear the error audio cue just one time in the first character "x" not in other characters causing the error.

## Expected Behavior

See how the audio cues in diff editor behaves. That's ideal! In diff editor audio cues are played even on a character-level when users use Left/RightArrow keys. This is fantastic because users can hear where the diff starts and ends precisely. However, the error/warning audio cues are not working like that.

## System Info

Version: 1.75.0-insider (system setup)
Commit: 07d6f5b35fac559504dec65e3073d796b2d25a55
Date: 2023-01-26T05:23:05.014Z
Electron: 19.1.9
Chromium: 102.0.5005.194
Node.js: 16.14.2
V8: 10.2.154.23-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes

**Comment by alexr00** [comment]
I've verified that the audio cue plays for each character of the error. However, it no longer plays when the cursor enters the line. Is this intentional/desirable?

**Comment by meganrogge** [comment]
Hmm, I cannot reproduce that @alexr00 . A gif would be helpful - moving from one line to another via up and down arrow keys still works for me. also using the mouse to go to the line works

**Comment by alexr00** [comment]
@meganrogge I have my cursor here:
![image](https://user-images.githubusercontent.com/38270282/220944925-530685ea-d4ab-4733-b0fb-ccc683cff2bc.png)

And then I hit `right arrow` to go to the new line.

**Comment by meganrogge** [comment]
if you right arrow there (or down arrow), you won't be on the arrow - you'll be on the semicolon, which is not a part of the error

**Comment by meganrogge** [comment]
Ah thanks for that clarification. @jooyoungseo would be the one to ask here. 

The test case @alexr00 described is:
- insert this code in an editor:

```
// this is a long comment
const x = _notDefined;
```
- put your cursor right after `comment`
- arrow right or down so the cursor lands to the right of the semi-colon
- there is an error specifically for the `_notDefined` range which does not include a semi-colon

the question is whether or not the error audio cue should be played.
